_opcodes/op000.html

```diff
@@ -19,7 +19,7 @@
  If the targets AC is already 'AC Modifier' or below, this effect will do nothing).
 
 <p>
-AC is capped to the range [-20, 20]
+Each modifier type to AC from this opcode is capped to the range [-20, 20]. Each AC type total is capped to the range [-32768,32767].
```

Comment:
Neither version is exact. AC as the base stat (likely the modifiers too, as you note) was capped to [-20,20], which then effects can modify arbitrarily.